docs/develop/api-reference/http-rest-api.md

```diff
@@ -0,0 +1,238 @@
+| Parameter&nbsp;name | Required?               | Type                 | Description                                                                                                                                                        |
+| -------------- |-------------------------|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| authorization | no | URL-safe&nbsp;string | The Momento auth token, in string format, is used for authentication/authorization of the request. Must be provided either here or as the `token` query parameter. |
+
+### Response
+#### Ok
+
+*Status Code: 204 No Content*
+
+- The key-value pair was stored in the cache, but no content was returned (by design).
+
+#### Error
+*Status Code: 400 Bad Request*
+- This error type typically indicates that the request was incorrectly specified. See the message body for further details.
+
+*Status Code: 401 Unauthorized*
+
+- This error type typically indicates that the Momento auth token passed in is either invalid or expired. See the body of the message for further details.
+
+*Status Code: 403 Forbidden*
+
+- This error type typically indicates the Momento auth token passed in does not grant the required access to the resources you attempted. See the body of the message for further details.*Status Code: 401 Unauthorized*
+
+- This error type typically indicates that the Momento auth token passed in is either invalid or expired. See the body of the message for further details.
+
+*Status Code: 403 Forbidden*
+
+- This error type typically indicates the Momento auth token passed in does not grant the required access to the resources you attempted. See the body of the message for further details.
+
+*Status Code: 404 Not Found*
+- “Cache Not Found” indicates that the specified cache does not exist.
+
+*Status Code: 429 Too Many Requests*
+- This error type typically indicates that account limits were exceeded. See the message body for further details, or contact Momento support to request a limit increase.
+
+*Status Code: 500 Internal Server Error*
+- This error type typically indicates that the service is experiencing issues. Contact Momento support for further assistance.
+
+## Examples
+### Put
+URL-safe Key, token as query parameter:
+
+`curl -X PUT -d 'example value' "https://api.cache.<endpoint>/cache/default-cache?key=example_key&ttl_seconds=300&token=<token>"`
+
+Base64 Key, token as header:
+`curl -X PUT -H "authorization: <token>" -d 'example value' "https://api.cache.<endpoint>/cache/default-cache?key_base64=ZXhhbXBsZV9rZXk&ttl_seconds=300"`
+### Get
+URL-safe key, token as query parameter
+
+`curl "https://api.cache.<endpoint>/cache/default-cache?key=example_key&token=<token>"`
+
+Base64 Key, token as header:
+
+`curl -H "authorization: <token>” "https://api.cache.<endpoint>/cache/default-cache?key_base64=ZXhhbXBsZV9rZXk"`
```

Comment:
Docs appears to have "helped" us by formatting one of these quotes as a unicode end-quote character (after <token>)

Reply:
Not sure I follow. Can you please fix it, commit it, and then merge this PR? I have to take my son to a DR appt.